app/frontend/src/components/ches/MergeForm.vue

```diff
@@ -200,18 +200,13 @@
         </v-col>
         <!-- preview button -->
         <v-col md="4">
-          <v-btn
-            width="100%"
-            large
-            outlined
-            @click="loadPreview()">
+          <v-btn width="100%" large outlined @click="loadPreview()">
```

Comment:
If I recall correctly, if you aren't passing a parameter into the function, you can just reference the function definition and don't have to in-line invoke it with the ().